src/modules/flow/max31855/max31855.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "max31855",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD 3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "input/hw",
+      "description": "MAX31855, A Temperature Reader",
+      "methods": {
+        "close": "max31855_close",
+        "open": "max31855_open"
+      },
+      "name": "aio/max31855",
+      "options": {
+        "members": [
+          {
+            "data_type": "int",
+            "description": "Bus",
+            "name": "bus"
+          },
+          {
+            "data_type": "int",
+            "description": "Chip Select",
+            "name": "chip_select"
+          }
+        ],
+        "version": 1
+      },
+      "in_ports": [
+        {
+          "data_type": "any",
+          "description": "Signal the SPI chip that a read must be performed",
+          "methods": {
+            "process": "max31855_process"
+          },
+          "name": "POLL_SIGNAL",
+          "required": false
+        }
+      ],
+      "out_ports": [
+        {
+          "data_type": "float",
+          "description": "Measured value.",
+          "name": "OUT"
```

Comment:
Measured value in which unity? On other temperature sensors, we use CELSIUS, FAHRENHEIT and even KELVIN out ports - please follow that.